src/modules/displayData.js

```diff
@@ -0,0 +1,37 @@
+import { getLikesArr, addLike } from './involvement-api.js';
+import cardsCounter from './cards-counter.js';
+import { showPopup } from './popup.js';
+
+const container = document.querySelector('.grid-container');
+const counterDisplay = document.querySelector('.counter');
+
+const displayCards = async (recipes) => {
+  const arrayOfLikes = await getLikesArr();
+  for (let i = 0; i < recipes.length; i += 1) {
+    let nroLikes = 0;
+    const likedObj = arrayOfLikes.find((element) => element.item_id === recipes[i].id);
+    if (likedObj) {
+      nroLikes = likedObj.likes;
```

Comment:
When the like button is clicked more than once times, it only shows one like. Kindly fix it 👍